_drafts/2017-09-20-refactoring-sensenet-redux-store.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Diving deeper into sensenet ECM's Redux store"
+author: herflis
+image: "/img/posts/sn_plus_redux.png"
+tags: [Redux, sensenet, JavaScript, TypeScript, React]
+---
+
+As you may already know [sensenet ECM](https://sensenet.com)'s UI is being fully rebuilt on a new stack including [Redux](http://redux.js.org/). Since sensenet is also a development platform with the possibility of building your own app upon its Redux store and actions, let me guide you through the current version of the state tree showing you how we organized data so far.
+
+---
+In the last few months I've read a couple of posts about the Redux store of Twitter and I thought if it was interesting to dig into their state tree it will be useful to write an article like this about sensenet ECM's too. [Dissecting Twitter's Redux Store](https://medium.com/statuscode/dissecting-twitters-redux-store-d7280b62c6b1),  [Diving Deeper into Twitter’s Redux Store: Adventures in Minified Vendor Javascript](https://medium.com/@nuncamind/diving-deeper-into-twitters-redux-store-adventures-in-minified-vendor-javascript-67fbac5dc219) and other posts in this topic were not only guide us analyzing sensenet ECM's Redux store the same way in a similar post, but also helped us to figure out how a state tree in a content-heavy application could be built-up, and how we should structure data in our next DMS MVP's Redux store.
```

Comment:
were not only guide us... -> were not only guiding us in analyzing...
the tense is not right here imho